src/uct/ib/base/ib_device.c

```diff
@@ -287,6 +287,30 @@ ucs_status_t uct_ib_device_init(uct_ib_device_t *dev,
         break;
     }
 
+    if (IBV_EXP_HAVE_ATOMIC_HCA(&dev->dev_attr) ||
+        IBV_EXP_HAVE_ATOMIC_GLOB(&dev->dev_attr) ||
+        IBV_EXP_HAVE_ATOMIC_HCA_REPLY_BE(&dev->dev_attr))
+    {
```

Comment:
maybe it is worth to memset(0) atomic_arg_size?

Reply:
Maybe it's even calloced (need to check). Typically we memseted only the relevant part of struct.
Need to agree on a common approach here

Reply:
here https://github.com/openucx/ucx/blob/master/src/uct/ib/base/ib_md.c#L1420